report_docx/__openerp__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# © <2016> <Siyuan Gu>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{'name': 'Report Docx',
+ 'version': '9.0.1.0.0',
+ 'category': 'report',
+ 'depends': ['base'],
```

Comment:
The engine module only depends on base, no other module.
It follows the idea from the OCA module report_xls

https://github.com/acsone/reporting-engine/blob/8.0/report_xls/